packages/kie-bc-editors-unpacked/pom.xml

```diff
@@ -0,0 +1,199 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2020 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~        http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>kie-bc-editors-unpacked</artifactId>
+    <groupId>org.kogito-tooling</groupId>
+    <packaging>jar</packaging>
+    <version>0.0.0</version>
+
+    <repositories>
+        <repository>
+            <id>JBoss Nexus</id>
+            <url>https://repository.jboss.org/nexus/content/groups/public</url>
+            <snapshots>
+                <enabled>true</enabled>
+                <updatePolicy>always</updatePolicy>
+            </snapshots>
+        </repository>
+    </repositories>
+
+    <properties>
+        <version.dmn.webapp>7.38.0-SNAPSHOT</version.dmn.webapp>
+        <version.bpmn.webapp>7.38.0-SNAPSHOT</version.bpmn.webapp>
+        <version.scesim.webapp>7.38.0-SNAPSHOT</version.scesim.webapp>
```

Comment:
We should discuss what's the best first version for the editors.

Reply:
For master, I think it's fine to keep the same as the community. As soon as we have a position for productization, we will need to branch/tag something and we change those properties accordingly.

Reply:
/cc @porcelli